### PR TITLE
RC2 Announcement Blog Post

### PR DESCRIPTION
**Pull request by rowanmiller**
This PR won't actually be merged, just using it as a way to review the
blog post before we publish

### CODE REVIEW

**Comment by rowanmiller** [comment]
cc @divega @richlander @danroth27 this is a draft of the EF Core RC2 blog post, feel free to provide any feedback. It's in a public repository since there is nothing confidential in here anymore and our provider writers may want to look it over.

File: Post [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+The situations where we would recommend using EF Core are:
+ * New applications that do not require features that are not yet implemented in EF Core
+ * Applications that target .NET Core, such as Universal Windows Platform (UWP) and ASP.NET Core applications
+
+For all other applications, you should consider using EF6.x. EF6.x will continue to be a supported release for some time – see the end of this post for more details on upcoming releases.
+
+Because of the fundamental changes in EF Core we do not recommend attempting to move an EF6.x application to EF Core unless you have a compelling reason to make the change.  If you want to move to EF Core to make use of new features, then make sure you are aware of its limitations before you start. **You should view the move from EF6.x to EF Core as a port rather than an upgrade.**
+
+
+Getting started with EF7
```
Comment (divega) [inline Post]: Should be Getting started with EF Core 1.0 :smile:

**Comment by divega** [comment]
LGTM (only a minor typo).

**Comment by richlander** [comment]
LGTM

**Comment by rowanmiller** [comment]
> Doesn't clearly state where to get EF Core.

The acquisition flow depends on the type of app you are writing (and the package you install depends on the provider you are targeting) so we tend to push folks to the appropriate docs for the platform they are targeting.

**Pull request closed without merging**